Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:18.7.0 as builder
+
+RUN mkdir -p /home/node/app
+WORKDIR /home/node/app
+COPY package*.json /home/node/app/
+RUN chown -R node:node /home/node/
+
+USER node
+
+RUN yarn install
```

Comment:
lets use frozen lockfile here

Reply:
Okey, thats why I asked you , can you please let me know , what exactly is needed for the install and build.

Reply:
yes, this should be adjusted